requirements.txt

```diff
@@ -7,3 +7,4 @@ mock
 dbus-python
 PyGObject
 pycairo
+caldav
```

Comment:
We're not using requirements anymore since we're 100% meson. Feel free to leave this change (since the file will be removed eventually), but it can't be depended on for dependencies.